pkgs/development/libraries/sundials/3.x.nix

```diff
@@ -12,11 +14,15 @@ stdenv.mkDerivation rec {
 
 
   cmakeFlags = [
-    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples"
+    "-DEXAMPLES_INSTALL_PATH=${placeholder "out"}/share/examples" ] ++
+     stdenv.lib.optional (lapackSupport) [ "-DSUNDIALS_INDEX_TYPE=int32_t"
+      "-DLAPACK_ENABLE=ON"
+      "-DLAPACK_LIBRARIES=[${liblapack}/lib/liblapack${stdenv.hostPlatform.extensions.sharedLibrary},${openblas}/lib/libopenblas${stdenv.hostPlatform.extensions.sharedLibrary},${gfortran.cc.lib}/lib/libgfortran.a]"
```

Comment:
does cmake support array notation? The "[" and "]" inside the string look a bit weird.

Reply:
I don't know what your comment means.